### ISSUE TITLE
Makefile:120: recipe for target 'sub-make' failed

### ISSUE THREAD

**Issue by ruCyberPoison** [firmadyne/kernel-v2.6.32#1]
@firmadyne
@ddcc 
Hello Devs,

Today i tried to compile a mipsel kernel with all files the repository give here but when i input the command:

`make ARCH=mips CROSS_COMPILE=/opt/cross/mipsel-linux-musl/bin/mipsel-linux-musl- O=./build/mipsel -j8`

also tried

`make ARCH=mips CROSS_COMPILE=/opt/cross/mipsel-linux-musl/bin/mipsel-linux-musl- O=./build/mipsel -j8 all`

I receive every time this error and i can't go to the next step cross-compilation 

`  LD      fs/built-in.o
Makefile:120: recipe for target 'sub-make' failed
make: *** [sub-make] Error 2
root@ns2:/Firmware/kernel-v2.6.32# `

`  LD      drivers/built-in.o
Makefile:120: recipe for target 'sub-make' failed
make: *** [sub-make] Error 2
root@ns2:/Firmware/kernel-v2.6.32# `

so it's possible to fixes this without reinstall the OS actually i use Ubuntu 17.04 with this headers
4.13.0-46-generic

Rgds,
David Martins - Screamfox

**Comment by ddcc** [comment]
Is this the same as https://github.com/firmadyne/firmadyne/issues/82 ?

**Comment by ruCyberPoison** [comment]
Yes is similar like [fimadyne/firmadyne#82](https://github.com/firmadyne/firmadyne/issues/82) 

Do you know how can i install gcc 5.3.0 and g++ 5.3.0 i googled but nothing found for my ubuntu version (17.04 -latest version of ubuntu 17.04 )

Rgds
